features/step_definitions/basic_steps.rb

```diff
@@ -36,3 +36,11 @@
   user = User.find_by(email: email)
   login_as(user, scope: :user)
 end
+
+Given(/^I am on the new booking page$/) do
+  visit '/bookings/new.2017-04-04'
```

Comment:
Does this route do what you think it does? Even if so, you should have this be `I am on the booking page for "2017-04-04"` -> don't hard-code values like this in generic-sounding steps.